### PR TITLE
Merge ENG-2238 changes into ENG-2287

### PR DESCRIPTION
**Pull request by raxkaynan**
Amend AttributeFields

### CODE REVIEW

**Review by fedelocci — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/400#pullrequestreview-664053752" title="Approved">fedelocci</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)~~ [fedelocci]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
